lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -412,7 +412,7 @@ def add_line_items(xml, options={})
               xml.tag! :PC3DtlIndex,  byte_limit(index + 1, 2)
               line_item.each do |key, value|
                 if key == :line_tot
-                  formatted_key = :PC3Dtllinetot
+                  formatted_key = :PC3DtlLineTot
```

Comment:
It looks to me like we can actually just get rid of this clause entirely, since `LineTot` = `:line_tot.to_s.camelize`?

Reply:
otherwise lgtm; same tests seem to fail for me on master as on branch. 👍